### DISCUSSION TITLE
custom playbook / sigma rule help

### DISCUSSION THREAD

**Discussion by Pancakesarusrex** [Security-Onion-Solutions/securityonion#5725] [Unsupported Versions] [answered]
i would like to create a playbook play that leads to an elastalert that accomplishes a similar result to this example. https://github.com/Yelp/elastalert/blob/master/example_rules/example_change.yaml

i want an alert when there is a change in a field while keeping the other field constant. i basically want to know when a user logs into multiple computers at once. i have set up an osquery that gets this info into kibana and now i have the fields. result.columns.user.security and result.hostname.security. what would be the sigma syntax to look at both fields and alert when there is a change on the result.hostname.security field. 

any help would be greatly appreciated.

**Comment by defensivedepth** [comment, marked as answer]
This type of Elastalert rule is not currently possible in Playbook, however this will all be re-evaluated once the new version of Sigma is released (https://github.com/SigmaHQ/pySigma) which should be soon!

**Comment by clairmont32** [comment]
You may be able to do this with a manual lucene or KQL search and make a visualization or table for it. Windows event logs would have the login info so if you see the same user with different hosts in an arbitrary timespan you'd be able to see that data.

**Reply by Pancakesarusrex** [reply]
do you know how i could lucene or KQL search for the arbitrary timespan. lets say if windows event 6424 within a span of 15 minutes in the last 24 hours?
